chips/lowrisc/Cargo.toml

```diff
@@ -11,3 +11,7 @@ edition.workspace = true
 [dependencies]
 rv32i = { path = "../../arch/rv32i" }
 kernel = { path = "../../kernel" }
+
+[lints.rust]
+# These are unused and unsupported
```

Comment:
```suggestion
TODO: this is a non-canonical use of `cfg` predicates. This may represent dead code that should simply be removed, or a different mechanism for choosing its inclusion, such as a feature flag or generics, should be used.
```